client/src/components/Modal/__tests__/Modal.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Text } from 'react-native';
+import renderer from 'react-test-renderer';
+
+import Modal from '../Modal';
+
+const noop = () => {};
+
+const child = (
+  <Text>test</Text>
+);
+
+test('renders correctly', () => {
+  const tree1 = renderer.create(
+    <Modal
```

Comment:
lol wut did you forget how to nest components?
```
<Modal title="test" closeModal={noop} visible>
  {child}
</Modal>
```

Reply:
ooh is that what it meant. duh.